src/drem/transform/vo.py

```diff
@@ -52,6 +70,15 @@ def _remove_null_address_strings(df: pd.DataFrame, on: str) -> pd.DataFrame:
     return df
 
 
+@task
+def _remove_zero_floor_area_buildings(df: pd.DataFrame) -> pd.DataFrame:
+
+    df = df[df["Area"] > 0]
+
+    return df
```

Comment:
```suggestion
    
    return df[df["Area"] > 0]
```
Why not do a one-liner here!?